spacify-portal/src/main/java/org/uci/spacifyPortal/controllers/UserController.java

```diff
@@ -0,0 +1,29 @@
+package org.uci.spacifyPortal.controllers;
+
```

Comment:
This too should be a part of the roomController as this API will called while operating with the room object